sys/include/fido2/ctap/ctap.h

```diff
@@ -142,10 +141,9 @@ extern "C" {
 /* set default button if no button is configured */
 #ifdef BTN0_PIN
 #define CTAP_UP_BUTTON BTN0_PIN
-/* if no button available disable UP test */
 #else
 #define CONFIG_FIDO2_CTAP_DISABLE_UP 1
-#endif
+#endif /* BTN0_PIN */
```

Comment:
What is actually the desired behavior? Silently disable user presence on boards that do not expose a pin? Or better throw a warning and exclude these boards from CI? Throw a warning here and disable user presence in the test by default?

Reply:
I think we had a discussion about this when I added these lines and we agreed on the first approach. 
Murdock complained about a symbol being redefined. I hopefully fixed this issue now and it should work like the first approach with the recent changes.

Reply:
Doesn't seem to be fixed. Maybe run the compile test locally before.

Reply:
Sorry. Didn't think about just trying to compile locally with the failing boards. Did that now

Reply:
Success :)